src/app/app.component.ts

```diff
@@ -26,15 +34,15 @@ export class AppComponent implements OnInit, AfterViewInit {
     appRef.isStable.subscribe((status: boolean) => {
       console.log(`The application is stable? ${status}`);
     });
-    if (environment.production) {
+    // if (environment.production) {
```

Comment:
This probably doesn't need to be commented it out as it isn't related to service workers.

Reply:
yes, I just comment it for local debug, I will add it back.